### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [flatCore/flatCore-CMS#89] [Q&A] [answered]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by patkon** [comment, marked as answer]
Hi codehangen, i will close this project soon. I've started a new one here (just a few minutes ago):
https://github.com/SwiftyEdit/SwiftyEdit
It is largely the same as flatCore. New Name, new features, cut off some old stuff.
**You are of course very welcome to join the project.**

**Reply by ghost** [reply]
Hi patkon.

> Hi codehangen, i will close this project soon. I've started a new one here (just a few minutes ago):
> https://github.com/SwiftyEdit/SwiftyEdit
> It is largely the same as flatCore. New Name, new features, cut off some old stuff.
> You are of course very welcome to join the project.

thank you for feedback. I want to contribute with SwiftyEdit and/or flatCore.
